Lib/test/test_json/test_decode.py

```diff
@@ -88,7 +108,7 @@ def test_string_with_utf8_bom(self):
             self.json.load(StringIO(bom_json))
         self.assertIn('BOM', str(cm.exception))
         # make sure that the BOM is not detected in the middle of a string
-        bom_in_str = '"{}"'.format(''.encode('utf-8-sig').decode('utf-8'))
+        bom_in_str = f"\"{''.encode('utf-8-sig').decode('utf-8')}\""
```

Comment:
What is the difference?

Reply:
re: https://github.com/python/cpython/pull/115693#discussion_r1539438377, f-string instead of `.format()`, shall I undo?

Reply:
In this case using `.format()` feels cleaner.

Reply:
Yup, the original is more readable here.
```suggestion
        bom_in_str = '"{}"'.format(''.encode('utf-8-sig').decode('utf-8'))
```

You could also do:
```suggestion
        bom = ''.encode('utf-8-sig').decode('utf-8')
        bom_in_str = f'"{bom}"'
```
or import `BOM_UTF8` from `codecs`.